packages/story-editor/src/components/panels/design/videoOptions/test/videoOptions.js

```diff
@@ -18,13 +18,20 @@
  * External dependencies
  */
 import { screen } from '@testing-library/react';
-
+import { useFeature } from 'flagged';
 /**
  * Internal dependencies
  */
 import VideoOptions from '../videoOptions';
 import { renderPanel } from '../../../shared/test/_utils';
 import { useLocalMedia } from '../../../../../app/media';
+import mockUseVideoElementTranscoding from '../../../../../app/media/utils/useVideoElementTranscoding';
```

Comment:
Unrelated, but why does `useVideoElementTranscoding` exist twice in the code base? We can delete one of the two.

Reply:
No idea. Out of scope of this work.